src/components/Project/Project.tsx

```diff
@@ -1,43 +1,43 @@
-import { CSSProperties } from "react";
-import { Add, Star } from "@material-ui/icons";
 import styled from "styled-components";
+import { Add, Star } from "@material-ui/icons";
+
+import ProjectDefinition from "../../interfaces/Project";
+
 import Button from "../Button";
 import IconButton from "../IconButton";
 import EditableLabel from "../EditableLabel";
+import Column from "../Column";
 
 export interface Props {
-  title: string;
-  children?: React.ReactNode;
-  className?: string;
-  style?: CSSProperties;
+  project: ProjectDefinition;
 }
 
-export default function Project({ title, children, ...props }: Props): JSX.Element {
+export default function Project({ project }: Props): JSX.Element {
   return (
-    <Root {...props}>
-      <Content>
-        <Header>
-          <EditableTitle value={title} />
-
-          <StarButton>
-            <Star />
-          </StarButton>
-        </Header>
-
-        <Panels>
-          {children}
```

Comment:
This component won't have anything but <Column /> as a direct child. So I figured removing `children` from this component and letting it have to render logic within so that I can add DnD logic there without exposing it to `App.tsx`which is two layers higher from Column's perspective.